Create Readme.md (#2)
glific/mobile: Readme.md

# Glific

## Getting Started

### Prerequisites
* Nodejs
* Android SDK

### Clone the repo in local machine switch to your branch.
- Run ``git clone https://github.com/glific/mobile.git``

> - Run git checkout -b <Add_YOUR_NAME>

Reviewer: Add this in the installation steps. Remove the checkout line. This is a fairly standard practice and need not to be included in readme.

### Steup locally

- ``cd moblie``
- Run ``npm -i install``
- If any warning are their then run ``npm audit fix --force``
- Run ``npm start``

### Open in Android Mobile

> - After this QR will be displayed
> - From Playstore download Expo Go
> - Then just scan the scanner.

Reviewer: ```suggestion
- Download Expo Go app from playstore
- Scan the QR from your CLI
```

- ![Login_page](https://github.com/Abhay-86/web-development/assets/76918911/baa0435c-3676-4c2f-998f-8340a0363209)

> - ![Server_page](https://github.com/Abhay-86/web-development/assets/76918911/782408b9-2354-4c27-a051-c3be6364bc4d)

Reviewer: We won't be needing these.
